README.md

```diff
@@ -36,15 +38,15 @@ before it.
 Support Window (Security)
 -------------------------
 
-- Critical security issues should be fixed in *two* versions behind the most
-recently released.
+- If critical security fixes are identified, the maintainer should apply
+them to the most recently released version and *two* versions before it.
 
 **Note** however that there is currently no quantified or documented
 commitment to tracking CVEs, nor any guarantee to apply all possible
-security fixes. While it is of course tracked to the best of the
-maintainer's ability, the project is dependent on community input, and all
-the _responsibility remains on the adopting companies_ to secure their
-final products.
+security fixes.  While it is of course tracked to the best of the
+maintainer's ability, the project is dependent on community input.  All the
+_responsibility remains on the adopting companies_ to secure their final
+products.
 
```

Comment:
Good clarifications in the update Gunnar.

Reply:
Confirmed today we are in agreement and to merge this.